cs/mailing.texy

```diff
@@ -163,23 +163,57 @@ V šabloně pak odkaz vytvoříme pomocí makra `{link}` tak, jak jsme zvyklí.
 <a href="{link Presenter:action}">Odkaz</a>
 \--
 
-Odesílání přes SMTP
-===================
+Typy mailerů
+============
 
-Výchozí mailer((třída zajišťující odesílání e-mailů)) používá PHP funkci [php:mail]. Pokud bychom potřebovali poštu odesílat skrze SMTP server, použijte `SmtpMailer`
+|* Mailer        |* Podpora SMTP |* Exception
+| SendmailMailer | Ne            | SendException
+| SmtpMailer     | Ano           | SmtpException
+| FallbackMailer | Ano           | FallbackMailerException
+
+Výchozí mailer((třída zajišťující odesílání e-mailů)) je SendmailMailer, který používá PHP funkci [php:mail].
+
+Můžeme si také vytvořit vlastní mailer, jedná se o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+
+
+SMTP Mailer
+--------------
+
+Pokud bychom potřebovali poštu odesílat skrze SMTP server, použijte `SmtpMailer`.
 
 /--php
-$mailer = new Nette\Mail\SmtpMailer([
+$mailer = new Nette\Mail\SmtpMailer(array(
         'host' => 'smtp.gmail.com',
         'username' => 'franta@gmail.com',
         'password' => '*****',
         'secure' => 'ssl',
-]);
+));
 $mailer->send($mail);
 \--
 
 
-Můžeme si také vytvořit vlastní mailer, jde o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+Fallback Mailer
+----------------
+
+Máme-li občas potíže s odesíláním emailů z jednoho maileru (např. výpadek smtp serveru), můžeme využít `FallbackMailer`, ve kterém nastavíme seznam mailerů pro případ, že předchozí selže.
```

Comment:
To „z jednoho maileru“ bych zrušil plus bych už v té úvodní větě zmínil, že to umí zkusit poslat ten mail opakovaně, tj. vznikne pak něco jako

Máme-li občas potíže s odesíláním emailů (např. kvůli výpadku SMTP serveru), můžeme využít `FallbackMailer`, který umí v případě chyby automaticky zopakovat pokus o odeslání buď pomocí stejného nebo pomocí alternativního maileru.